#
# old_revision []
#
# add_dir ""
# 
# add_file "a"
#  content [9decd90bc2eedb78c74d5059a6bb6244dd2d0562]
#
============================================================
--- /dev/null	
+++ a	9decd90bc2eedb78c74d5059a6bb6244dd2d0562
@@ -0,0 +1,2 @@
+abc
+abc
